tests/test_channel_properties.py

```diff
@@ -0,0 +1,197 @@
+"""Tests for validating channel properties for different basic data types."""
+import random
+
+import pytest
+
+import nidaqmx
+from nidaqmx.constants import (
+    BridgeConfiguration,
+    CounterFrequencyMethod,
+    Edge,
+    ExcitationSource,
+    FrequencyUnits,
+    PowerIdleOutputBehavior,
+    RTDType,
+    ResistanceConfiguration,
+    TemperatureUnits,
+    TerminalConfiguration,
+    TimeUnits,
+    VoltageUnits,
+)
+from nidaqmx.tests.helpers import generate_random_seed
+
+
+class TestChannelPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for different
+    data types of channel properties.
+    """
```

Comment:
Remove the test class and use test functions.

Reply:
Removed test class